opampserver/pkg/sdkconfig/configsections/instrumentationconfig.go

```diff
@@ -0,0 +1,45 @@
+package configsections
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/odigos-io/odigos/api/odigos/v1alpha1"
+	"github.com/odigos-io/odigos/opampserver/protobufs"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func GetWorkloadInstrumentationConfig(ctx context.Context, kubeClient client.Client, configObjectName string, ns string) (*v1alpha1.InstrumentationConfig, error) {
+	instrumentationConfig := &v1alpha1.InstrumentationConfig{}
+	err := kubeClient.Get(ctx, client.ObjectKey{Namespace: ns, Name: configObjectName}, instrumentationConfig)
+
+	if err != nil && !apierrors.IsNotFound(err) {
+		return nil, err
+	}
+
+	return instrumentationConfig, nil
+}
+
+func FilterRelevantSdk(instrumentationConfig *v1alpha1.InstrumentationConfig, programmingLanguage string) (*protobufs.AgentConfigFile, error) {
+	relevantSdkConfig := v1alpha1.SdkConfig{}
+
+	for _, sdkConfig := range instrumentationConfig.Spec.SdkConfigs {
```

Comment:
What if we do not find and sdkConfig for this language? should we return an empty config or error?